Tests/RenderingTests/assertMatch.swift

```diff
@@ -34,6 +35,7 @@ extension XCTestCase {
         environment: @escaping (inout EnvironmentValues) -> () = { _ in },
         size: CGSize? = nil,
         lifetime: XCTAttachment.Lifetime = .deleteOnSuccess,
+        useDrawingGroup: Bool = true,
```

Comment:
This should not be the default imo. There are many other Views that are backed by UIKit/AppKit that don't crash. Instead, you end up with an image like this: 
<image src="https://github.com/liveview-native/liveview-client-swiftui/assets/13581484/f6fe7817-2e5f-4ac2-954f-6a4cbf74bf61" width="100" />

All of the List tests are now essentially comparing this image to itself. There's no documentation on if/when Apple will switch a View to/from being backed by a platform view. So we should only enable this if we know it is necessary for a test.

Reply:
Oh, that's not good. I'll switch the default.

Reply:
@carson-katri Please see the main PR description for how I addressed this.